### PR TITLE
[ONNX] minor doc improvements and cleanup

### PR DESCRIPTION
**Pull request by garymm**
* Fix some bad formatting and clarify things in onnx.rst.
* In `export_to_pretty_string`:
    * Add documentation for previously undocumented args.
    * Document that `f` arg is ignored and mark it deprecated.
* Use double quotes for string literals in test_operators.py.

### CODE REVIEW

File: torch/onnx/__init__.py [hunk 1]
```diff
@@ -310,9 +310,24 @@ def forward(self, x):
                         custom_opsets, enable_onnx_checker, use_external_data_format)
 
 
-def export_to_pretty_string(*args, **kwargs):
+def export_to_pretty_string(*args, **kwargs) -> str:
     r"""
-    Same as :func:`export`, but returns a text representation of the exported model.
+    Similar to :func:`export`, but returns a text representation of the ONNX
+    model. Only differences in args listed below. All other args are the same
```
Comment (jiafatom) [inline torch/onnx/__init__.py:316]: nit: We have some works on deprecating arguments in `export`. Need keep the args in `export` and `export_to_pretty_string` in sync.

**Review by jiafatom — APPROVED** [review]
LGTM

File: torch/onnx/__init__.py (hunk 1 above)
Comment (garymm) [reply torch/onnx/__init__.py:337]: Thanks for catching that. Done!

**Comment by garymm** [comment]
Check failures seem unrelated to my change, so merging.

**Pull request merged**